pkg/dogstatsd/server_test.go

```diff
@@ -591,6 +590,9 @@ func TestDebugStatsSpike(t *testing.T) {
 	time.Sleep(500 * time.Millisecond)
 	// it is no more considered a spike because we had another second with 500 metrics
 	assert.False(s.hasSpike())
+
+	s.Stop()
```

Comment:
Maybe more idiomatic to put these on `defer` statements near where they're instantiated? Feel free to ignore.